### DISCUSSION TITLE
BUG of saving by changing icon in the menu image area

### DISCUSSION THREAD

**Discussion by davijls9** [mattieserver/netbox-topology-views#389] [Q&A] [answered]
How to play:

1 - Update from version 3.2.1 to version 3.6.0
2 - Leave only the netbox-topologia-views plugin
3 - Go to the netbox-topology-views menu and access image transfer
4 - Change any item and try to save.
Gives the error:
😢 {"error": "'str' object has no attribute 'removeprefix'", "exception": "AttributeError", "netbox_version": "3.6.2", "python_version": "3.8.10"}

I expected that when choosing the icon corresponding to that topic/item of choice, it would change and save.

The behavior is not to save and gives an error. Error: 😢 {"error": "'str' object has no attribute 'removeprefix'", "exception": "AttributeError", "netbox_version": "3.6.2", "python_version": "3.8.10"}

![Captura de tela 2023-09-27 080858](https://github.com/mattieserver/netbox-topology-views/assets/132774090/feb3fb5a-67f8-4251-9889-359fa9b973e4)
![Captura de tela 2023-09-27 080749](https://github.com/mattieserver/netbox-topology-views/assets/132774090/5431ac30-b116-4700-92df-a77b0207c68b)

Version: v3.6.2 and v3.6.0 NETBOX, tested on both. Could you help me resolve this problem?

**Comment by mattieserver** [comment, marked as answer]
This is a known issue:https://github.com/mattieserver/netbox-topology-views/issues/375
